GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -455,63 +446,41 @@ public static string CloneCmd(string fromPath, string toPath)
         /// <para><c>NULL</c>: don't pass any such param to git.</para>
         /// </param>
         /// <param name="lfs">True to use the <c>git lfs clone</c> command instead of <c>git clone</c>.</param>
-        public static string CloneCmd(string fromPath, string toPath, bool central, bool initSubmodules, [CanBeNull] string branch, int? depth, bool? isSingleBranch, bool lfs)
+        public static string CloneCmd(string fromPath, string toPath, bool central = false, bool initSubmodules = false, [CanBeNull] string branch = "", int? depth = null, bool? isSingleBranch = null, bool lfs = false)
         {
             var from = PathUtil.IsLocalFile(fromPath) ? fromPath.ToPosixPath() : fromPath;
-            var to = toPath.ToPosixPath();
-            var options = new List<string> { "-v" };
-            if (central)
-            {
-                options.Add("--bare");
-            }
-
-            if (initSubmodules)
-            {
-                options.Add("--recurse-submodules");
-            }
-
-            if (depth.HasValue)
-            {
-                options.Add("--depth " + depth);
-            }
-
-            if (isSingleBranch.HasValue)
-            {
-                options.Add(isSingleBranch.Value ? "--single-branch" : "--no-single-branch");
-            }
-
-            options.Add("--progress");
-            if (branch == null)
-            {
-                options.Add("--no-checkout");
-            }
-            else if (branch != "")
-            {
-                options.Add("--branch " + branch);
-            }
 
-            options.Add(string.Format("\"{0}\"", from.Trim()));
-            options.Add(string.Format("\"{0}\"", to.Trim()));
-
-            var command = lfs ? "lfs clone " : "clone ";
+            var args = new ArgumentBuilder
+            {
+                { lfs, "lfs" },
```

Comment:
💭 This is short, but I'm 100% convinced it's better

Reply:
Well I see I totally messed up my comment by forgetting 'not'

Reply:
I wondered whether it was the case... :)